externs/olx.js

```diff
@@ -2366,6 +2366,21 @@ olx.interaction.DrawOptions.prototype.condition;
 
 
 /**
+ * @typedef {{features: (ol.Collection.<ol.Feature>|undefined)}}
+ * @api
+ */
+olx.interaction.TranslateOptions;
+
+
+/**
+ * The features the interaction works on.
```

Comment:
I think this requires more information, to indicate that this option is not mandatory, and what the behavior is when it is not specified.